config/rbac/csi_rbd_provisioner_clusterrole.yaml

```diff
@@ -45,6 +45,9 @@ rules:
   - apiGroups: ["snapshot.storage.k8s.io"]
     resources: ["volumesnapshotcontents/status"]
     verbs: ["update", "patch"]
+  - apiGroups: ["groupsnapshot.storage.k8s.io"]
+    resources: ["volumegroupsnapshotclasses"]
+    verbs: ["create", "delete", "get", "list", "watch"]
```

Comment:
Need to update RBAC for both cephfs and rbd like we did in this PR https://github.com/rook/rook/pull/13832/files. we don't need create/delete access for cephcsi